[2016-04-29 08:42:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /_profiler/"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /_profiler/\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:06:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /_profiler/phpinfo"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /_profiler/phpinfo\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:07:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /_profiler/phpinfo"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /_profiler/phpinfo\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:07:24] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /_profiler/"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /_profiler/\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:46:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:47:43] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:51:17] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 10:55:22] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 15:29:22] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 15:58:26] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 15:59:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 16:00:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 16:00:24] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 16:03:17] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-04-29 16:03:25] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello/dirk"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello/dirk\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-05-01 17:42:39] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /router:debug"" at /vagrant/app/cache/prod/classes.php line 2449 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /router:debug\" at /vagrant/app/cache/prod/classes.php:2449, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /vagrant/app/cache/prod/appProdUrlMatcher.php:53)"} []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:39] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\EventDispatcher\Debug\WrappedListener::__invoke". [] []
[2016-05-01 17:42:40] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". [] []
[2016-05-01 17:42:40] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". [] []
[2016-05-01 17:42:40] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". [] []
[2016-05-01 17:42:40] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelController". [] []
[2016-05-01 17:42:40] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". [] []
[2016-05-01 17:42:42] event.DEBUG: Listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException" stopped propagation of the event "kernel.exception". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\TranslatorListener::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\Security\Http\Firewall::onKernelFinishRequest". [] []
[2016-05-01 17:42:42] event.DEBUG: Notified event "kernel.terminate" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onTerminate". [] []
